app/src/main/java/org/simple/clinic/summary/linkId/LinkIdWithPatientSheetController.kt

```diff
@@ -10,7 +10,7 @@ import org.simple.clinic.patient.PatientRepository
 import org.simple.clinic.widgets.UiEvent
 import javax.inject.Inject
 
-typealias Ui = LinkIdWithPatientSheet
+typealias Ui = LinkIdWithPatientView
 typealias UiChange = (Ui) -> Unit
 
 class LinkIdWithPatientSheetController @Inject constructor(
```

Comment:
Please rename the class`LinkIdWithPatientSheetController` to `LinkIdWithPatientViewController`

Reply:
Done.